app/src/main/kotlin/com/wire/android/ui/common/textfield/WireTextField.kt

```diff
@@ -117,6 +115,67 @@ internal fun WireTextField(
     }
 }
 
+@Composable
+internal fun BorderLessWireTextField(
```

Comment:
"borderless" is a valid single word, L shouldn't be a capital letter 😄

Wouldn't be better to just pass the custom `WireTextFieldColors` with transparent border color instead of creating a copy of the whole WireTextField just to hide the border?